kedro/contrib/idempotent/idempotent_runner.py

```diff
@@ -74,6 +115,7 @@ class IdempotentSequentialRunner(AbstractRunner):
 
     def __init__(self):
         super().__init__()
+        self.force_run = False
         self.state_storage = IdempotentStateStorage()
```

Comment:
Move this into the `_run` function so that we can get access to the `catalog`

Reply:
And pass the `catalog` into the `IdempotentStateStorage` so that we can look for the dedicated catalog.

Add the special name as an attribute on to the class.
`IDEMPOTENT_STATE_STORAGE_CATALOG_NAME = 'idempotent_state_storage'`

Reply:
> Add the special name as an attribute on to the class.
 Not as a variable in the module